app/scripts/controllers/paymentModal.js

```diff
@@ -326,6 +326,7 @@ angular.module('confRegistrationWebApp')
     /*
     Returns false is payment if payment and refund channel are credit card and current time is less than 24 hours from
     payment time.  If a refund is issued before the payment settles, authorize.net will do a full refund regardless of amount.
+    ***Authorize.net no longer used, but payment is still used so keeping code below for now.***
     */
     $scope.isPartialRefundAvailable = function(payment, refundChannel) {
```

Comment:
I think the restriction imposed by Authorize.NET, namely to not allow refunds (full or partial) within a certain period of time after the initial payment, does not apply at all to TSYS. I think that if the refund request is initiated within a certain window of time, say where the transaction has not yet settled, then the transaction will be voided. Otherwise, the transaction will be returned. 

Therefore, any ERT code which applies the no longer applicable business rule can be removed.